examples/src/file_streaming_defaults.py

```diff
@@ -0,0 +1,79 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example shows how to use the streaming encrypt and decrypt APIs when working with files.
+
+For the purposes of this example, we demonstrate using AWS KMS,
+but you can use other key management options with the AWS Encryption SDK.
+Look in the ``keyring`` and ``master_key_provider`` directories
+for examples that demonstrate how to use other key management configurations.
```

Comment:
For X, see Y.
For examples that demonstrate how to use other key management configurations, see the ``keyring`` and ``master_key_provider`` directories